pages/write/[[...roomId]].js

```diff
@@ -84,7 +47,7 @@ export default function Write() {
     const [validatePassword, setValidatePassword] = useState('');
     const [actionSelect, setActionSelect] = useState('');
     const [serviceSpaceId, setServiceSpaceId] = useState();
-    const [openActions, setOpenActions] = useState(false);
+    const [, setOpenActions] = useState(false);
```

Comment:
This is also something that's going to always be there when we make use of the action menu. So it should live inside there...

To do this we could pass every "action" that's supposed to be rendered with the corresponding `ServiceSubmenu.Item` like this:

```js
// this replaces the `renderSelectedOption()` method inside of `write/index.js`
const ActionNewAnonymousPad = (<form onSubmit={(e) => { e.preventDefault(); createAnonymousPad(); }}>
                    <input type="text" placeholder={t('pad name')} value={newPadName} onChange={(e) => setNewPadName(e.target.value)} />
                    <button type="submit" disabled={!newPadName}>{ loading ? <LoadingSpinner inverted /> :t('Create pad') }</button>
</form>);

...

<ServiceSubmenu.Item actionComponentToRender={ActionNewAnonymousPad}>{ t('Create new anonymous pad') }</ServiceSubmenu.Item>
```